src/input.rs

```diff
@@ -68,8 +68,8 @@ fn parse_range(input: &str) -> Result<PortRange, String> {
 /// Fast Port Scanner built in Rust.
 /// WARNING Do not use this program against sensitive infrastructure since the
 /// specified server may not be able to handle this many socket connections at once.
-/// - Discord https://discord.gg/GFrQsGy
-/// - GitHub https://github.com/RustScan/RustScan
+/// - Discord <https://discord.gg/GFrQsGy>
```

Comment:
```suggestion
/// - Discord <http://discord.skerritt.blog>
```

Annoyingly we have updated our Discord, and also there is no SSL on that domain _yet_ as I have run out of CLoudflare page-rules 🙈 it is on my todo list